keybase_proofs/views.py

```diff
@@ -151,6 +151,13 @@ def post(self, request, *args, **kwargs):
                 kb_proof.is_verified = valid_proof
                 kb_proof.sig_hash = sig_hash
                 kb_proof.save()
-                return redirect(self.get_success_url())
+                self.success_args = {
+                    'kb_ua': kb_ua,
+                    'kb_username': kb_username,
+                    'sig_hash': sig_hash,
+                    'username': request.user.username,
+                    'domain': get_domain(),
+                }
+                return redirect(self.get_success_url(), permanent=True)
```

Comment:
do we need `success_args` in `self` instead of passing them to `get_success_url` ?

Reply:
i'll change this. `get_success_url(self)` is a method of the class but we can define our own here instead https://docs.djangoproject.com/en/2.1/ref/class-based-views/mixins-editing/#django.views.generic.edit.FormMixin.get_success_url